dotCMS/src/main/java/com/dotmarketing/portlets/languagesmanager/action/EditLanguageAction.java

```diff
@@ -143,8 +143,15 @@ private void _save(ActionRequest req, ActionResponse res, PortletConfig config,
 		Language language = (Language) req.getAttribute(WebKeys.LANGUAGE_MANAGER_LANGUAGE) ;
 		BeanUtils.copyProperties(language,form);
         if (UtilMethods.isSet(language.getLanguageCode()) && UtilMethods.isSet(language.getLanguage())) {
-			try{
-				languageAPI.saveLanguage(language);
+			try {
```

Comment:
the Validation that if the lang is duped should be in the API and add a test

Reply:
and probably surface an AlreadyExistException

Reply:
The idea is do not touch the language api to avoid issues on PP